docs/collections/documents/tutorials/bulk-update-restql.md

```diff
@@ -206,15 +224,54 @@ const run = async function () {
 
         await executeRestql("updateData", {
             updateKeys,
-            updateKeyValue,
+            updateKeyValue
         });
 
         console.log("Data updated \n");
 
         const dataAfterUpdate = await executeRestql("getData");
         console.log(dataAfterUpdate.result);
         console.log("\n");
+
+        /* ------------------------ Saving a Restql Query ----------------------- */
```

Comment:
I am not sure about that I thought whenever we called queries via APIs we called it RestQl. In case RestQL term is not supposed to be used here then I guess we can change the title to Bulk Update with Query Worker??

Reply:
Anyways I have removed the RestQl word and just kept it as Saving a query, Executing a query, Updating a query , etc